build/dependency.sh

```diff
@@ -81,6 +81,8 @@ cat >"${DEP_PR}"<<EOF
 
 EOF
 
+$MVN dependency:resolve
```

Comment:
is it possible to combine it w/ line 35? just like 

```
$MVN dependency:resolve dependency:build-classpath -pl ...
```

Reply:
Several reasons to do it separately,
1.  better not to do in one place as not sure whether the change will impact or contaminate the outputs for dependencyList.
2. solely resolving maven dependency and the output will be more clear to read and to know which process we are waiting for (eg. resolving deps , building_classpath)